app/controllers/job_posts_controller.rb

```diff
@@ -0,0 +1,85 @@
+class JobPostsController < ApplicationController
+  before_filter :fix_dates, only: [:create, :update]
+  load_and_authorize_resource except: [:index]
+
+  def index
+    if current_school.present?
+      @job_posts = JobPost.where(school: current_school)
+                   .active.order("ends_at ASC")
+      @title = "All jobs for #{current_school.name}"
+    else
+      @job_posts = JobPost.active.order("ends_at ASC")
+      @title = "All job listings"
+    end
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @job_posts }
+    end
+  end
+
+  def show
+  end
+
+  def new
+  end
+
+  def edit
+  end
+
+  def create
+    respond_to do |format|
+      if @job_post.save
+        format.html { redirect_to @job_post,
+                      notice: 'Job post was successfully created.' }
+        format.json { render json: @job_post, status: :created,
+                             location: @job_post }
+      else
+        format.html { render action: "new" }
+        format.json { render json: @job_post.errors,
+                             status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @job_post.update_attributes(job_post_params)
+        format.html { redirect_to @job_post,
+                      notice: 'Job post was successfully updated.' }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 49, 68 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.